### ISSUE TITLE
508-defect-1 ❗ Launch-blocker [KEYBOARD]: All functionality MUST be accessible via keyboard

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#9949]
# [508-defect-1 ❗ Launch-blocker](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-1)

**Feedback framework**
- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

On page load of the 1 of 3 Veteran information, Shipping address, screen, using the keyboard to navigation, it is not possible to access the Edit permanent address option. Keyboard focus moves to:

1. Send my order to this address
1. Email address input field

The actual options available for a user to choose for shipping on this screen are:

1. Select permanent address on file (as shown) 
1. Edit permanent address on file 
1. Select temporary address, if on file
1. Add a temporary address to ship to

The current design conceals those choices, and results in a inaccessible and confusing experience for assistive technology users. 

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a keyboard user, I want to access all the options available to me.

## Environment

* Operating System: all
* Browser: all
* Server destination: staging

## Steps to Recreate

1. Enter https://staging.va.gov/hearing-aid-batteries-and-accessories/veteran-information/addresses in browser, logged in as 228
2. Tab through the interface
3. Verify that the keyboard is unable to access either Edit permanent address or Add a temporary address

## Possible Fixes (optional)

Clarify the design so that the distinct shipping choices are presented:

1. Select permanent address on file
1. Edit permanent address on file 
1. Select temporary address, if on file
1. Add a temporary address to ship to

## WCAG or Vendor Guidance (optional)

* [W3C WCAG Keyboard: Understanding SC 2.1.1](https://www.w3.org/TR/UNDERSTANDING-WCAG20/keyboard-operation-keyboard-operable.html) - All functionality of the content is operable through a keyboard interface without requiring specific timings for individual keystrokes, except where the underlying function requires input that depends on the path of the user's movement and not just the endpoints. (Level A)

**Comment by 1Copenut** [comment]
Temporary address is not able to receive keyboard focus either. It looks like these are links without `href` attributes.

**Comment by bradl3yC** [comment]
@jenstrickland can you validate in staging?

**Comment by jenstrickland** [comment]
Validated. 🎉
